tests/object.spec.ts

```diff
@@ -55,11 +57,30 @@ describe("Dictionary", () => {
       .with(m => m.DictionaryProp,
         new Schema<DictionaryPropModel>()
           .with(x => x.DictionaryChildStringProp, /^[A-z]+\.[A-z]+$/)
+          .with(x => x.DictionaryChildNumberProp, x => x > 100)
       )
       .build();
 
-    assertInvalid(schema, model);
+    const errors = assertInvalid(schema, model);
+    expect(errors.length).to.be.eq(2);
```

Comment:
It might also be good to validate that the errors are what you expect them to be as well?